lrc/Pos/Lrc/Fts.hs

```diff
@@ -194,6 +198,27 @@ whenever we transition to the next coin, its index bona fide exceeds the
 previous upper bound (and thus it's more or equal to the current lower bound).
 
 -}
+
+-- | Choose several random stakeholders (specifically, their amount is
+-- currently hardcoded in 'Pos.Constants.epochSlots'). It is important to
+-- note that the only source of random-ness in this compuation is the shared
+-- random seed passed in as the first parameter.
+--
+-- The probability that a stakeholder will be chosen is proportional to the
+-- number of coins this stakeholder holds. The same stakeholder can be picked
+-- more than once.
+--
+-- How the algorithm works: we sort all unspent outputs in a deterministic
+-- way (lexicographically) and have an ordered sequence of pairs
+-- @(StakeholderId, Coin)@. Then we choose several random 'i's between 1 and
+-- amount of satoshi in the system; to find owner of 'i'th coin we find the
```

Comment:
I guess the atomic unit is not called "satoshi" in Cardano.

Reply:
The smallest indivisible unit of Ada is called the "Lovelace". Not sure that really relevant information here.

Reply:
It's just that « between 1 and the amount of satoshi in the system » should really be « between 1 and the amount of lovelace in the system ».

Reply:
Fixed, but I wonder at what point do we rename this to `followTheLovelace`? :)